catalog/ec2-policies.yaml

```diff
@@ -10,73 +10,51 @@
     - test: "StringNotLike"
       variable: "ec2:InstanceType"
       values:
-        - a1
-        - a1.metal
-        - c5
-        - c5.metal
-        - c5a
-        - c5ad
-        - c5d
-        - c5d.metal
-        - c5n
-        - c5n.metal
-        - c6g
-        - c6g.metal
-        - c6gd
-        - c6gd.metal
-        - c6gn
-        - d3
-        - d3en
-        - g4
+        - a1.*
+        - c5.*
+        - c5a.*
+        - c5ad.*
+        - c5d.*
+        - c5n.*
+        - c6g.*
+        - c6gd.*
+        - c6gn.*
+        - d3.*
+        - d3en.*
+        - g4.*
         - i3.metal
```

Comment:
why `i3.metal` is left, but `i3en.metal` is replaced with `i3en.*` ?

Reply:
should we replace all `.metal` with `.*` ?

Reply:
The list reflects the [AWS documentation](
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-types.html#ec2-nitro-instances). Virtualized `i3en` is listed, but `i3` is not. 

> The following instances are built on the Nitro System:
>
>   - Virtualized: A1, C5, C5a, C5ad, C5d, C5n, C6g, C6gd, C6gn, D3, D3en, G4, I3en, Inf1, M5, M5a, M5ad, M5d, M5dn, M5n, M5zn, M6g, M6gd, p3dn.24xlarge, P4, R5, R5a, R5ad, R5b, R5d, R5dn, R5n, R6g, R6gd, T3, T3a, T4g, X2gd, and z1d
>
>   - Bare metal: a1.metal, c5.metal, c5d.metal, c5n.metal, c6g.metal, c6gd.metal, i3.metal, i3en.metal, m5.metal, m5d.metal, m5dn.metal, m5n.metal, m5zn.metal, m6g.metal, m6gd.metal, mac1.metal, r5.metal, r5b.metal, r5d.metal, r5dn.metal, r5n.metal, r6g.metal, r6gd.metal, u-6tb1.metal, u-9tb1.metal, u-12tb1.metal, u-18tb1.metal, u-24tb1.metal, x2gd.metal, and z1d.metal